applications/main/nfc/helpers/protocol_support/nfc_protocol_support.c

```diff
@@ -489,6 +500,10 @@ static const NfcProtocolSupportCommonSceneBase
                 .on_event = nfc_protocol_support_scene_info_on_event,
                 .on_exit = nfc_protocol_support_scene_info_on_exit,
             },
+        [NfcProtocolSupportSceneCardDump] =
+            {.on_enter = nfc_protocol_support_scene_card_dump_on_enter,
+             .on_event = NULL,
+             .on_exit = nfc_protocol_support_scene_card_dump_on_exit},
```

Comment:
Please, add ',' after `nfc_protocol_support_scene_card_dump_on_exit` and run linter: ./fbt format

Reply:
Done